src/react/ui-elements/MiddleEllipsis.jsx

```diff
@@ -0,0 +1,117 @@
+//@flow
```

Comment:
Cool component, should we move it to core-ui? @Cuervino 
If yes, we could do it in a following PR @theocerutti.

Reply:
Just one thing : I added "width" props in MiddleEllipsis for a workaround.
Indeed, MiddleEllipsis doesn't work fine in a AutoSizer of react-window.
That's why MiddleEllipsis component needs to rerender if AutoSizer sizes has changed.